<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnEdit</class>
 <widget class="QWidget" name="ColumnEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>316</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ColumnEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RealCompare.qrc">
    <normaloff>:/Resources/edit/global/ndd.ico</normaloff>:/Resources/edit/global/ndd.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="textGroupBox">
       <property name="title">
        <string>Insert Text</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="inputText">
          <property name="maxLength">
           <number>1024</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="numGroupBox">
     <property name="title">
      <string>Insert Num</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Initial value:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="initNum">
          <property name="text">
           <string>1</string>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
          <property name="placeholderText">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>increment:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="incNum">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Repetitions:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="repeNum">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QCheckBox" name="addPrefix">
          <property name="text">
           <string>prefix string:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="prefix">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Format</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QRadioButton" name="is10">
             <property name="text">
              <string>Decimal </string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="is16">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="capital">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Capital</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QRadioButton" name="is8">
             <property name="text">
              <string>Octal </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="is2">
             <property name="text">
              <string>Binary</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RealCompare.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>inputText</sender>
   <signal>returnPressed()</signal>
   <receiver>ColumnEdit</receiver>
   <slot>slot_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>ColumnEdit</receiver>
   <slot>slot_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColumnEdit</receiver>
   <slot>slot_insertTextEnable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>ColumnEdit</receiver>
   <slot>slot_insertNumEnable(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>ColumnEdit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_ok()</slot>
  <slot>slot_insertTextEnable(bool)</slot>
  <slot>slot_insertNumEnable(bool)</slot>
 </slots>
</ui>
